tests/test_generate_multi_issues_file.py

```diff
@@ -0,0 +1,33 @@
+import sys
+
+import repobee
+from repobee_feedback._generate_multi_issues_file import (
+    MULTI_ISSUES_FILENAME,
+    GENERATE_MULTI_ISSUES_FILE_ACTION,
+    GenerateMultiIssuesFile,
+)
+
+
+class TestGenerateMultiIssuesFile:
+    """Tests generation of a multi-issues file"""
+
+    def test_creates_non_empty_output_file(self, tmp_path):

```

Comment:
My bad, parts of this test were actually a little bit broken, see comments. It doesn't matter terribly much as the test is just here to ensure that we get any output at all, but it's still nice to have sensible input.